wiki/support/modding-with-phone.md

```diff
@@ -0,0 +1,117 @@
+---
+sidebar: auto
+---
+# Quest modding with Android
+## Installing BMBF with an Android Phone
+
+* [Requirements](#requirements)
+* [Enable Developer mode on your Quest](#how-to-enable-developer-mode-on-the-oculus-quest)
+* [Setup your Phone](#setup-your-phone)
+* [Installing BMBF with your Phone](#installing-bmbf-with-your-phone)
+* [Setup Beat Saber](#setup-beat-saber)
+
+#### Requirements
+
+* An Android Phone or Android Tablet (IPhones or IPads are not supported)
+* A **paid** Version of Beat Saber on the Oculus Quest Store
+* A Cable to connect your Quest to your Phone (If your phone charges over USB C the charger that was included with your
+Quest should work)
+
+
+#### How to enable developer mode on the Oculus Quest
+First head to [the oculus dashboard](https://dashboard.oculus.com/) and sign in with your Facebook Account.
+You'll need to provide a credit card to verify.
+If you don't have a credit card you can do the steps [here](#i-have-no-credit-card).
+After you successfully logged in and verified you can create your new Organization.
+
+![DevModeDashboard](https://github.com/ComputerElite/wiki/blob/main/BM/DevModeDashboard.png)
+
+![DevModeCreateOrg](https://github.com/ComputerElite/wiki/blob/main/BM/DevModeCreateOrg.png)
+
+After you created it, you can enable Developer mode in the oculus app on your smartphone.
+
+##### Enable Developer mode in the oculus app
+
+![EnableDevMode](https://github.com/ComputerElite/wiki/blob/main/BM/EnableDevMode.png)
+
+#### I have no credit card
+[Original Source](https://www.reddit.com/r/sidequest/comments/jaxy4u/cant_verify_oculus_developer_account/?utm_source=amp&utm_medium=&utm_content=post_body).
+
+So please note that you have to create a second account (oculus developer account) and then add  your facebook profile to the organization created.
+It is a bit harder.
+
+**info:** If you have a friend who has a organization you can ask them to add you to their organization so you don't have to make a new account.
+If so tell your friend to check [Add yourself to the organization](#add-yourself-to-the-organization)
+
+##### Create a new developer account
+Go to the [sign up page for oculus developers](https://developer.oculus.com/sign-up/) and click "create a unmerged oculus developer account".
+Then Fill in all needed information.
+Then verify with the email you got sent and log in again with the account you just created.
+Then go to the [verify page](https://developer.oculus.com/manage/verify/) and set up Two Facto Authentication. 
+
+![2fa](https://github.com/ComputerElite/wiki/blob/main/BM/2fa.png)
```

Comment:
should put images into the local assets folder. Path is `wiki\.assets\images\beginners-guide` with referencing as `~@images/beginners-guide/filename.png` 

Note there is a image file size limit of 1mb / image so should reduce quality as appropriate.

Reply:
Images should be there now and they are 1 MB in total so that should be good